pkg/kudoctl/cmd/init/webhook.go

```diff
@@ -0,0 +1,160 @@
+package init
+
+import (
+	"fmt"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"github.com/kudobuilder/kudo/pkg/kudoctl/clog"
+	"github.com/kudobuilder/kudo/pkg/util/kudo"
+	"k8s.io/api/admissionregistration/v1beta1"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	clientv1beta1 "k8s.io/client-go/kubernetes/typed/admissionregistration/v1beta1"
+	"sigs.k8s.io/yaml"
+)
+
+// Install uses Kubernetes client to install KUDO manager prereqs.
+func installWebhook(client kubernetes.Interface, dynamicClient dynamic.Interface, ns string) error {
```

Comment:
This is more like `installInstanceValidatingWebhook` - the current name suggests it is generic where it is actually pretty specific.